setup.py

```diff
@@ -25,11 +25,10 @@
     package_data={"fourmat": ("assets/*.*", "assets/.*")},
     install_requires=(
         "click>=7",
-        # Pin these to avoid unplanned messy diffs.
-        "black==20.8b1",
-        "flake8-bugbear==20.1.4",
-        "flake8==3.8.4",
-        "isort~=5.5",
+        "black>=20,<21",
```

Comment:
Revert to pinning black for now as suggested

```suggestion
        "black==20.8b1",
```